src/coreclr/crossgen-corelib.proj

```diff
@@ -75,7 +75,7 @@
     </PropertyGroup>
     
     <PropertyGroup Condition="$(BuildPdb)">
-      <CrossGenPdbCmd>$(DotNetCli) $(BinDir)\r2rdump\r2rdump.dll</CrossGenPdbCmd>
+      <CrossGenPdbCmd>$(BinDir)\r2rdump\r2rdump.exe</CrossGenPdbCmd>
```

Comment:
Is this going to use some random .NET runtime installed on the machine? That does not sound good.

Reply:
Hmm, I don't see any trivial way to fix this as part of this PR. I'm going to revert this last commit and I'll tackle this as part of the switch-over of SPC to use CG2, probably by prioritizing my in progress change to enable building the PDB directly from Crossgen2, that should remove the need to run r2rdump from CoreCLR build.

Reply:
Reverted in 5th commit.